content/departments/product-engineering/engineering/code-graph/search/guiding_principles.md

```diff
@@ -0,0 +1,41 @@
+
+Our job is to help provide context and clarity on direction: what problems are we seeking to solve as a team, as a company? Why do these problems matter? What are north star outcomes? As you absorb this direction, and understand how it relates to your domain of expertise, you will be empowered to directly identify and break down the work that should happen to make progress towards north star outcomes. As you wrap up a work item, you should feel empowered to ask yourself: "what should I work on next? why?", and to bring your answers to the team. In your first few months, you may find that we end up asking you these questions ("what do _you_ think you should work on next?"), to encourage this practice.
+
+## Your scope can be arbitrarily wide
+
+While you may naturally gravitate towards the types of problems you have the most expertise solving, and while we will pick initial tasks that match your expertise, you do not need special permission to tackle problems outside of this scope. A practical example could be a backend-leaning engineer tackling frontend-leaning tasks. However, scope extends beyond the technical realm here: you should feel empowered to, for instance, suggest improvements to our team processes, challenge or clarify our team strategy, or bring forward opportunities to collaborate with other teams.
+
+We trust you to leverage this freedom with discernment, keeping good forward momentum towards the outcomes you, the team and the company are targeting. We can help you identify the best strategies to do so.
+
+## Keep plenty of breathing room
+
+We have found the team to be most effective when teammates keep plenty of breathing room: the team is more relaxed, has room for creativity, the quality of our work improves, and we end up making faster overall progress towards roadmap outcomes. Breathing room or slack time is traditionally hard to quantify, but as a ballpark idea try to cap roadmap commitments to 50-60% of your total bandwidth. We trust you with how you employ the rest of your bandwidth.
+
+You may find yourself using this breathing room to
+
+- Absorb an occasional underestimated task without straining your schedule
+- Be able to pursue codebase health improvements that you identify as you work towards roadmap outcomes
+- Grow your skills and scope
+- Develop and explore ideas you're passionate about
+- Or, sometimes, simply to breathe.
+
+Having and using this much breathing room may require an adjustment. Don't hesitate to talk to your teammates about their experience organizing their time and bandwidth.
```

Comment:
```suggestion
It may take time to adjust to having more breathing room in your schedule. Don't hesitate to talk to your teammates about their experience organizing their time and bandwidth.
```

"may require an adjustment" has just a tad of a "you need an adjustment/correction in your behavior" rather than "you may need time to adjust" (verb/process of adjusting).